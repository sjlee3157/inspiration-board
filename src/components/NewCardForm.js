import React, { Component } from 'react';
import PropTypes from 'prop-types';
import emoji from 'emoji-dictionary';

import './style/NewCardForm.css';

const EMOJI_LIST = ["", "heart_eyes", "beer", "clap", "sparkling_heart", "heart_eyes_cat", "dog"]

// This is a mock. The HTML is probably incorrect and will need to be changed.
const NewCardForm = (props) => {
  return (
    <section className="new-card-form">
      <section className="new-card-form__header">
      </section>
      <form className="new-card-form__form">
        <label className="new-card-form--label" htmlFor="text">Text</label>
        <textarea className="new-card-form--textarea" name="text"></textarea>
        <label className="new-card-form--select" htmlFor="emoji">Emoji</label>
        <select name="emoji"></select>
        <input className="new-card-form--form-button" type="submit" name="submit" value="Create A New Card!" />
      </form>
    </section>
  )
}

NewCardForm.propTypes = {

};

export default NewCardForm;
