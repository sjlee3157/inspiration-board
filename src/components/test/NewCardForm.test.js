import React from 'react';
import { shallow } from 'enzyme';
import NewCardForm from '../NewCardForm';

describe('NewCardForm', () => {

  it('will match the NewCardForm Snapshot', () => {
    const wrapper = shallow( <NewCardForm /> );
  expect(wrapper).toMatchSnapshot();
  });

});
