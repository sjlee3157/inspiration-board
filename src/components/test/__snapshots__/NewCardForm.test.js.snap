// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCardForm will match the NewCardForm Snapshot 1`] = `
<section
  className="new-card-form"
>
  <section
    className="new-card-form__header"
  />
  <form
    className="new-card-form__form"
  >
    <label
      className="new-card-form--label"
      htmlFor="text"
    >
      Text
    </label>
    <textarea
      className="new-card-form--textarea"
      name="text"
    />
    <label
      className="new-card-form--select"
      htmlFor="emoji"
    >
      Emoji
    </label>
    <select
      name="emoji"
    />
    <input
      className="new-card-form--form-button"
      name="submit"
      type="submit"
      value="Create A New Card!"
    />
  </form>
</section>
`;
